doc/whats_new.rst

```diff
@@ -254,7 +254,7 @@ New features
    - The new module :mod:`nilearn.connectome` now has class
      :class:`nilearn.connectome.ConnectivityMeasure` can be useful for
      computing functional connectivity matrices.
-   - The function :func:`nilearn.connectome.sym_to_vec` in same module
+   - The function :func:`nilearn.connectome.sym_matrix_to_vec` in same module
```

Comment:
you may need to revert it the same ```sym_to_vec``` and instead add in section 0.2.6 about these API changes ```sym_to_vec``` to ```sym_matrix_to_vec``` and new features ```vectorize``` and ```inverse_transform``` and example renaming with this PR. Whatever might be interesting to know about this PR can be added in 0.2.6